tests/test_rsa_keys.py

```diff
@@ -154,6 +157,31 @@ def test_verify_rsa_signature(self):
           scheme, public_rsa, data))
 
 
+  def test_verify_rsa_pss_sha256(self):
+    global public_rsa
+    global private_rsa
+
+    rsa_scheme = 'rsassa-pss-sha256'
+    data = 'The quick brown fox jumps over the lazy dog'.encode('utf-8')
+
+    private_key_object = load_pem_private_key(private_rsa.encode('utf-8'),
+        password=None, backend=default_backend())
+    digest_obj = securesystemslib.hash.digest_from_rsa_scheme(rsa_scheme,
+        'pyca_crypto')
+    max_salt_length = padding.calculate_max_pss_salt_length(private_key_object, digest_obj)
+
```

Comment:
`max_salt_length` should stay the same for this key, right? Does it make sense to assert the value, so that we know how many times `assertEqual` is actually called below?

Reply:
I don't think it'll test much other than Python FOR loops work as expected, and we didn't accidentally misuse it.

Please take a look at the new test to see if it makes you feel better?

Reply:
I might have [more comprehensive tests](https://github.com/hashicorp/vault/pull/16549). Pls let me know how you'd like to proceed. Thx.

Reply:
I'm actually fine the way it was before and don't fully understand the ["better test"](https://github.com/secure-systems-lab/securesystemslib/pull/422/commits/3f3ed970a45be47209714f5aa66fc10b4e299421) you added. See previous comment.